codalab/lib/codalab_manager.py

```diff
@@ -341,6 +341,12 @@ def default_user_info(self):
         info['disk_quota'] = formatting.parse_size(info['disk_quota'])
         return info
 
+    def launch_new_worker_system(self):
```

Comment:
I suppose if this is just a temporary flag that will be removed and always be true eventually, then we should write that and call it whatever we want.